com.unity.perception/Runtime/GroundTruth/PerceptionCamera.cs

```diff
@@ -74,6 +74,10 @@ public void AddScriptableRenderPass(ScriptableRenderPass pass)
         static ProfilerMarker s_FlipY = new ProfilerMarker("Flip Y (PerceptionCamera)");
         static ProfilerMarker s_EncodeAndSave = new ProfilerMarker("Encode and save (PerceptionCamera)");
 
+#if URP_PRESENT
+        bool m_GroundTruthRendererFeatureRun;
```

Comment:
Not sure what the convention is, but I generally use f for flags as in m_fGroundTruthRendererFeatureRun, and I've seen is/has etc throughout code before.  

Also, does it make sense to put this in an IFDEF vs. using a nullable variable for example?  Not a strong opinion here - since it's just a flag.

Reply:
The m_ is per our standard conventions for private members. Besides m_/k_ we aren't actually doing hungarian notation :) 
I've removed the IFDEFs everywhere except the final check. I agree it's cleaner this way